src/tests/run.proj

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="Directory.Build.props" />
```

Comment:
Since it looks like `run.proj` doesn't import the NoTargets SDK, it won't import the Directory.Build.props/targets files. If that's intentional, it might be worthwhile to move into a subdirectory and use the NoTargets SDK + empty Directory.Build.props/targets files to make it more easily understood at a glance. (I know I've sunk a few hours here and there trying to figure out why Directory.Build.props/targets wasn't being imported when I think it should have been or vice versa).

Reply:
I've been actually thinking about merging the run.proj script into the build.proj script; with this change it's no longer called directly, build.proj is always the "front-end"; after all, even with my additions build.proj has less than 200 lines and in combination with run.proj it will have just about 1000 lines. But I would also prefer to do that separately to make git history cleaner, merging the scripts will be a purely mechanical change.

Reply:
Sure we can hold off on that until later.